app/components/date-time-format-filter/date-time-format-filter.js

```diff
@@ -3,22 +3,30 @@
 angular.module('bulbs.cms.dateTimeFilter', [
   'bulbs.cms.site.config',
   'lodash',
-  'moment'
+  'moment',
 ])
   .filter('dateTimeFormat', [
     '_', 'moment', 'CmsConfig',
     function (_, moment, CmsConfig) {
-      return function (date, format) {
+      function isInvalidDateValue (dateValue) {
+        return !(
```

Comment:
I changed it to several negations. I just needed this function to be tolerant of `Date` values. Previously it assumed the passed in value was a moment object or a date string. The changes allow you to pass a valid date, date string, or moment object and get the same format behaviour.